README.md

```diff
@@ -1 +1,56 @@
-# Practice
\ No newline at end of file
+##Project Title
```

Comment:
Can you update the titles of each of the sections to proper markdown? I see the `##` is next to the `Project Title` for example and that prevents it from being rendered as a heading.

Reply:
Done, new commit added to resolve this issue